src/components/v2v3/V2V3Project/V2V3FundingCycleSection/settingExplanations.tsx

```diff
@@ -4,106 +4,155 @@ import { helpPagePath } from 'utils/routes'
 
 export const DISTRIBUTION_LIMIT_EXPLANATION = (
   <Trans>
-    The maximum amount of funds that can be distributed from the treasury in a
-    funding cycle.
+    The amount of ETH which can be paid out from this project during the cycle.
+    Payouts reset each cycle.
   </Trans>
 )
 
 export const DISCOUNT_RATE_EXPLANATION = (
   <Trans>
-    The project token's issuance rate decreases by the{' '}
-    <strong>discount rate</strong> every funding cycle. A higher discount rate
-    incentivizes contributors to fund the project earlier.
+    The issuance rate is reduced by this amount each cycle (unless the owner
+    changes the issuance rate).
   </Trans>
 )
 
 export const REDEMPTION_RATE_EXPLANATION = (
   <Trans>
     <p>
-      The redemption rate determines the amount of overflow each token can be
-      redeemed for at any given time.
+      Supporters can burn their tokens to reclaim some of the ETH not needed for
```

Comment:
I wonder if throughout, maybe `reserved for` is slightly more nice than `needed for`. Loose thought, non blocking!

Reply:
Will keep thinking. I don't like reserved because it's too close to "reserved rate". Maybe "not being used for" or "not necessary for"?